interface/ceed-basis.c

```diff
@@ -155,6 +158,87 @@ int CeedBasisCreateTensorH1Lagrange(Ceed ceed, CeedInt dim, CeedInt ncomp,
   return 0;
 }
 
+/**
+  @brief Create a non tensor product basis for H^1 discretizations
+
+  @param ceed       A Ceed object where the CeedBasis will be created
+  @param topo       Topology of element, e.g. hypercube, simplex, ect
+  @param dim        Topological dimension
+  @param ncomp      Number of field components (1 for scalar fields)
+  @param ndof       Total number of nodes
+  @param nqpts      Total number of quadrature points
+  @param interp1d   Row-major nqpts × ndof matrix expressing the values of nodal
```

Comment:
name for this and below. Look at the `make doc` warnings.

Reply:
make doc is happy now